### DISCUSSION TITLE
#15. VATZ biweekly meeting at `2022-08-17 08:00:00 -06:00:00`

### DISCUSSION THREAD

**Discussion by xellos00** [dsrvlabs/vatz#223] [General]
### 1. Overall
![image](https://user-images.githubusercontent.com/6308023/184049014-5f512e23-fdbd-44c3-97d6-d1465330f034.png)

### 2. Statistic Rate
Sprint | Issue fulfillment | progress rate(%)
--: | :--: | :--:
13|  7/20 | 35
14|  13/23 | 43

> Thanks for your hard work for 2 weeks.
> The target progress rate for each sprint is 40% to 55%. 
> YES!!

---

- Finished
  - #216 
  - #176
  - #127
  - #217 
  - #206 
  - #213
  - #211 
  - #208
  - #203
  - #199
  
- Move to Sprint 13
  - #197 
  - #85
  - #87
  - #119
  - #120 
  - #121 
  - #122 
  - #146 
  - #174 
  - #175  
  - #178 
  - #187
  - #202 

---
### discussion
- https://github.com/dsrvlabs/vatz/issues/233
- #225

**Comment by xellos00** [comment]
# 동요

---
- https://github.com/dsrvlabs/vatz/issues/224
- https://github.com/dsrvlabs/vatz/issues/225
- https://github.com/dsrvlabs/vatz/issues/229

**Comment by rootwarp** [comment]
# Joonkyo
- https://github.com/dsrvlabs/vatz/issues/230
- https://github.com/dsrvlabs/vatz/issues/221
- https://github.com/dsrvlabs/vatz/issues/222

**Comment by gnongs** [comment]
# Jinhong
- https://github.com/dsrvlabs/vatz/issues/226

**Comment by heejin-github** [comment]
## 희진
- https://github.com/dsrvlabs/vatz-plugin-solana/pull/21
- https://github.com/dsrvlabs/vatz-plugin-solana/pull/22

**Comment by kim201212** [comment]
## 형기

- no issue

**Comment by meetrick** [comment]
# 황제
- no issue
